getXls.js

```diff
@@ -0,0 +1,104 @@
+import puppeteer from 'puppeteer';
```

Comment:
getDailyReports.js could be better after analyzing the things happening in this file